mypy_self_check_strict.ini

```
@@ -20,4 +20,10 @@ always_false = MYPYC
 plugins = misc/proper_plugin.py
 python_version = 3.6
 exclude = mypy/typeshed/|mypyc/test-data/|mypyc/lib-rt/
-disallow_redefinition = True
\ No newline at end of file
+disallow_redefinition = True
+
+[mypy-mypy]
+default_return = True
+
+[mypy-mypyc]
+default_return = True
```

what `mypy` and `mypyc` means, can't it just be like
```suggestion
default_return = True
```

---

I only want it to apply to the mypy and mypyc packages, not globally.